src/ng/services/dom/table.js

```diff
@@ -9,6 +10,26 @@ export default class Table {
 		this._head = null;
 		this._body = null;
 		this._foot = null;
+		this.canExecute = this.canExecute.bind(this);
+	}
+
+	canExecute() {
+		const markup = this.markup;
+		const target = markup.table;
+		let current = markup.document.activeElement;
+		while (current) {
+			if (current === target) {
+				return true;
+			}
+
+			current = current.parentNode;
+		}
+
+		return false;
+	}
+
+	get keyDown() {
+		return new EventListener(this, this.markup.document);
 	}
```

Comment:
hmmm, it should return off method not listener. or
you need to refactor that so user can write